typescript/sdk/src/utils/cosmos.ts

```diff
@@ -9,6 +9,7 @@ export const CosmosChainSchema = z
       .regex(new RegExp('^(\\.\\./)+chain\\.schema\\.json$'))
       .optional(),
     chain_name: z.string().regex(new RegExp('[a-z0-9]+')),
+    chain_type: z.string().regex(new RegExp('[a-z0-9]+')),
```

Comment:
dang so zod complained when there was an extra field that was supplied without the schema knowing about it? very annoying. Wonder if there's a way to be tolerant of the data being a superset of the type as that feels safe for our purposes here

Reply:
That makes sense, will investigate